src/BitStringAddresses/bosefs.jl

```diff
@@ -266,6 +266,11 @@ struct BoseFSIndex<:FieldVector{3,Int}
     offset::Int
 end
 
+function Base.show(io::IO, i::BoseFSIndex)
+    @unpack occnum, mode, offset = i
+    print(io, "BoseFSIndex(occnum=$occnum, mode=$mode, offset=$offset)")
+end
+
```

Comment:
```suggestion
Base.show(io::IO, ::MIME"text/plain", i::BoseFSIndex) = show(io, i)

```
Found this method for `FermiFSIndex`, which seems to make sure that the above `show` method is actually used in the repl. This change may require fixing doctests.

Reply:
fixed